documentation/tutorials/creating_synthetic_tokens.md

```diff
@@ -0,0 +1,144 @@
+# Creating Synthetic Tokens from Truffle console
+
+This tutorial will show you how to create synthetic tokens from the command line using UMA’s synthetic token template. Before beginning this tutorial, please make sure your environment is set up correctly by following the instructions in the “Prerequisites” section. After completing this section, your environment will: 
+* Clone the UMA repo
+* Run Ganache on localhost port 9545
+* Compile all contracts in the UMA repo
+ 
+Below, we’ll discuss how to create and manage a token sponsor position via the command line on a local testnet. Please look here for information on existing UMA-supported testnet and mainnet deployments. 
+ 
+## Step 1: Deploy a contract and create tokens. 
```

Comment:
RC, can we change the titles to either all read "step 1, 2, 3" or none say this? (I think I prefer none). Can you also make sure the titles have consistent formatting (All Caps, or just first), and that the titles all flow together cohesively.

Reply:
Yes - I made all these changes in the google doc but have been slower to make them in github.